apps/pages/templates/pages/review_guide.html

```diff
@@ -83,7 +80,7 @@ <h2 id="tips">{{ _('Some tips for writing a great review') }}</h2>
     <ul>
       <li>
         {% trans %}
-          Submit one-word reviews such as "Great!", "wonderful," or "bad."
+          Submit one-word reviews such as "Great!", "Wonderful", or "Bad".
```

Comment:
Yes, this is basically a revert of 63fafaae8d87ba8ed6cabc42766488ff75d75068 but the way it currently is makes no sense:
* `wonderful,` Why would the example have a comma at the end but the enumeration of the examples misses it?
* `bad.` It's debatable whether the example should have punctuation or not, in any case, the punctuation for the whole sentence is missing.